<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalIntervalSeries_Widget</class>
 <widget class="QWidget" name="DigitalIntervalSeries_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Total Intervals:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="total_interval_label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="create_interval_layout">
       <item>
        <widget class="QPushButton" name="create_interval_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Create Interval</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_interval_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="start_frame_label">
         <property name="styleSheet">
          <string>QLabel { 
            font-weight: bold; 
            color: #0066CC;
            padding: 4px;
            border: 1px solid #CCCCCC;
            border-radius: 3px;
            background-color: #F0F8FF;
          }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="remove_interval_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove Interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flip_single_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Flip Single Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extend_interval_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Extend Interval To Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="merge_intervals_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Merge Selected Intervals</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_control_panel">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </item>
 <item>
  <widget class="Section" name="export_section" native="true">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="exportSectionLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="filenameLayout">
      <item>
       <widget class="QLabel" name="label_filename">
        <property name="text">
         <string>Output Filename:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filename_edit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="exportTypeLayout">
      <item>
       <widget class="QLabel" name="label_export_type">
        <property name="text">
         <string>Export Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="export_type_combo">
        <item>
         <property name="text">
          <string>CSV</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="exportTypeSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked_saver_options">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="CSVIntervalSaver_Widget" name="csv_interval_saver_widget"/>
     </widget>
    </item>
   </layout>
  </widget>
 </item>
 <item>
  <spacer name="verticalSpacer_bottom">
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <property name="sizeHint" stdset="0">
    <size>
     <width>20</width>
     <height>10</height>
    </size>
   </property>
  </spacer>
 </item>
</layout>
</widget>
 <customwidgets>
  <customwidget>
   <class>CSVIntervalSaver_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/DigitalTimeSeries/CSV/CSVIntervalSaver_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Section</class>
   <extends>QWidget</extends>
   <header>Collapsible_Widget/Section.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
